learning-team/2023/2023-10-23.md

```diff
@@ -0,0 +1,46 @@
+# Month Day, Year
+
+Facilitator & Note Taker: 
+
+## Attendees
+
+Add yourself to the list if you attend and check the box!
+
+### Core Team
+
+- [ ] Chris Manson (CM)
+- [ ] Jared Galanis (JG)
+- [ ] Jen Weber (JW)
+- [ ] Ricardo Mendes (RM)
+- [ ] Robert Wagner (RW)
+- [ ] Chris Ng (CN)
+- [ ] Anne-Greeth Schot-van Herwijnen (AG)
+
+### Guests
+
+- [] Your name here (INITIALS)
+
+## Weekly review
+
+[Open pull requests](https://help-wanted.emberjs.com/pull-requests)
```

Comment:
https://github.com/ember-learn/ember-website/pull/1059 would you be able to review